charts/humio-operator/templates/operator-rbac.yaml

```diff
@@ -191,6 +194,20 @@ rules:
   - get
   - list
   - watch
+{{- if and .Values.operator.pdfRenderService (eq .Values.operator.pdfRenderService.enabled true) }}
```

Comment:
We no longer need this conditional since we're not enabling/disabling via the helm chart right?